Protecs/src/main/java/uk/ac/ucl/protecs/sim/LoadPopulation.java

```diff
@@ -115,8 +121,11 @@ public static void load_population(String agentsFilename, WorldBankCovid19Sim si
 						w,
 						sim
 						);
+				if (usingWorkplaces) {
+					w.addPerson(p);
```

Comment:
Discussion about updating the "personsHere" list of the workplace re: does this introduce errors into presence/absence

Reply:
Conclusion: we may want to add a special holder to the Workplace object to make it easier to keep track of employees etc, but for the time being need to be sure that people's locations aren't accidentally set in two separate places when they're initialised!